### PR TITLE
Delete PocketMine stuff unneeded

### PR DESCRIPTION
**Pull request by erkam2002**

### CODE REVIEW

**Comment by MasterOfTheRealm** [comment]
Why take out PlayerAddOpEvent?

**Comment by erkam2002** [comment]
Its not exist..

**Comment by MasterOfTheRealm** [comment]
We can put it off as a future project. if we keep this PR on hold till that is done it might take a while

**Comment by Pizzahaet** [comment]
@MasterOfTheRealm it shouldn't take as long as you're making it seem, the entire GenisysPro website/forums could use as rewrite if anything, but yeah should take long at all

File: src/pocketmine/CrashDump.php [hunk 1]
```diff
@@ -123,33 +121,6 @@ private function pluginsData(){
 		}
 	}
 
-	private function extraData(){
```
Comment (Leo3418) [inline src/pocketmine/CrashDump.php]: I doubt if this deletion affects crash dump generation. More testing is required.

Comment (DerddyDert) [inline src/pocketmine/CrashDump.php]: He had other changes that are not in this PR. I think he needs to look over everything again and do a fresh PR.

**Pull request closed without merging**